src/views/Home/Home.js

```diff
@@ -7,12 +7,16 @@ import './Home.css';
 
 export default function Home() {
   // add useState calls here for title, subtitle, font, align, and text
-
+  const [title, setTitle] = useState('afef');
+  const [subtitle, setSubtitle] = useState('afef');
+  const [font, setFont] = useState('comforter');
+  const [align, setAlign] = useState('');
+  const [text, setText] = useState('aff');
   return (
     <main>
       {/* pass the state variables as props to the presentational components */}
-      <Preview />
-      <Editor />
+      <Preview {...{ title, subtitle, font, align, text }} />
+      <Editor {...{ setAlign, setFont, setSubtitle, setText, setTitle }}/>
```

Comment:
Nice work Alex! Great job spreading the props onto the components in Preview and Editor! Super clean code!